enumerables.rb

```diff
@@ -0,0 +1,137 @@
+module Enumerable
+  def my_each
+    i = 0
+    arr = self
+    if block_given?
+      while i < arr.count
+        yield(arr[i])
+        i += 1
+      end
+    else
+      arr
```

Comment:
- [ ] When no block is given my_each should return an Enumerator. 
Pay attention, an Array and an Enumerator are different classes, they are not the same.
Also remember that my_each could also work with a Range.
Make sure you return an Enumerator in your else, even if the seld if an Array or Range.
https://ruby-doc.org/core-2.6/Enumerator.html